conf/ocsci/flexy_baremetalpsi_upi.yaml

```diff
@@ -0,0 +1,8 @@
+DEPLOYMENT:
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetalpsi'
+  deployment_type: 'upi'
+
+FLEXY:
+  LAUNCHER_VARS: {"num_workers":"3", "openstack_network":"provider_net_cci_9","rhel_vm_template":"ql-rhel-7.6","qe_additional_repos":"[{'id': 'rhel7', 'name': 'rhel7', 'baseurl': 'http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os', 'enabled': 1, 'gpgcheck': 0}]","vm_type_masters":"m1.large","vm_type_workers":"ci.m1.xlarge.ocs03","ssh_key_name":"openshift-dev","rhel_ansible_user":"cloud-user","installer_payload_image":"registry.svc.ci.openshift.org/ocp/release:4.3","num_nodes": 1}
```

Comment:
@shylesh does this LAUNCHER_VARS needs to be in json format?
I guess it is but for our configuration it's not good for merging purpose with other config file if needed as it will overwrite whole LAUNCHER_VARS data.

I am thinking about to have those variables defined in pure yaml here, to allow us to merge data together from multiple config files and then when those data are used from here, transfer them to json.

so the yaml data will look like (just converted your data to yaml via python):

```yaml
installer_payload_image: registry.svc.ci.openshift.org/ocp/release:4.3
num_nodes: 1
num_workers: '3'
openstack_network: provider_net_cci_9
qe_additional_repos: '[{''id'': ''rhel7'', ''name'': ''rhel7'', ''baseurl'': ''http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os'',
  ''enabled'': 1, ''gpgcheck'': 0}]'
rhel_ansible_user: cloud-user
rhel_vm_template: ql-rhel-7.6
ssh_key_name: openshift-dev
vm_type_masters: m1.large
vm_type_workers: ci.m1.xlarge.ocs03
```

```
In [13]: yaml_data = yaml.dump(data, default_flow_style=False)

In [14]: print(yaml_data)
installer_payload_image: registry.svc.ci.openshift.org/ocp/release:4.3
num_nodes: 1
num_workers: '3'
openstack_network: provider_net_cci_9
qe_additional_repos: '[{''id'': ''rhel7'', ''name'': ''rhel7'', ''baseurl'': ''http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os'',
  ''enabled'': 1, ''gpgcheck'': 0}]'
rhel_ansible_user: cloud-user
rhel_vm_template: ql-rhel-7.6
ssh_key_name: openshift-dev
vm_type_masters: m1.large
vm_type_workers: ci.m1.xlarge.ocs03


In [15]: python_data = yaml.load(yaml_data)


In [16]: print(json.dumps(python_data))
{"installer_payload_image": "registry.svc.ci.openshift.org/ocp/release:4.3", "num_nodes": 1, "num_workers": "3", "openstack_network": "provider_net_cci_9", "qe_additional_repos": "[{'id': 'rhel7', 'name': 'rhel7', 'baseurl': 'http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os', 'enabled': 1, 'gpgcheck': 0}]", "rhel_ansible_user": "cloud-user", "rhel_vm_template": "ql-rhel-7.6", "ssh_key_name": "openshift-dev", "vm_type_masters": "m1.large", "vm_type_workers": "ci.m1.xlarge.ocs03"}
```

What do you think? Does not have to be in this PR, but we can open issue for it if you think it is worth.

Maybe our current functionality will cover this merging as well as it will be loaded as dictionary anyway, but for me the pure yaml format in the config is more readable than this one.

Reply:
ack,https://github.com/red-hat-storage/ocs-ci/issues/2489